tasklib/backends.py

```diff
@@ -95,13 +95,15 @@ class TaskWarrior(Backend):
     VERSION_2_4_5 = six.u('2.4.5')
 
     def __init__(self, data_location=None, create=True,
-                 taskrc_location='~/.taskrc', task_command='task'):
-        self.taskrc_location = os.path.expanduser(taskrc_location)
+                 taskrc_location=None, task_command='task'):
```

Comment:
why cancel the default `taskrc_location` while the original one works fine on windows?

Reply:
Does this mean that users will always need to specify a `rc` file by themself?